### ISSUE TITLE
Problem with Chime Volume

### ISSUE THREAD

**Issue by RobWP73** [dgreif/ring#816]
Hi.

First of all, this is an amazing plugin and has really helped with my Ring setup.

I have one problem, which might be a bug, or a user error!

I have had the plugin setup on Homebridge in two locations now, and in both cases, the volume of the chimes in the home has been stuck on 11 per cent. I can increase the volume again via the official Ring app, but when the device (e.g. doorbell) is triggered in Home, it reverts back to 11 per cent. both in Home and in the Ring app.

Am I missing something?

Thanks in advance for any help.

**Comment by dgreif** [comment]
Is it possible you have some automation set up to set the volume?  Are you still having this issue?

**Comment by RobWP73** [comment]
Hi Dusty

Sorry for not updating this ticket, but I worked out a few weeks later that the issue was what I had been trying to do, which Ring has not yet enabled. I thought I was able to set the volume of the chime and the bell separately (i.e. quieter notifications of motion and louder notifications of doorbell pushes). That’s not allowed. But because I was setting the volume for motion chimes and then exiting settings, I had not noticed it had then automatically reduced the volume of the bell by the same amount!

Kind regards.

Rob

Rob Westwood-Payne

Mob: 07958 912224

Resurgam, 4 Pittard Road,
Basingstoke, Hampshire, RG21 8YD

www.facebook.com/RobWestwood-Payne
www.twitter.com/RobWP73
www.instagram.com/RobWP73
